.gitignore

```diff
@@ -0,0 +1,6 @@
+/node_modules
+*.swp
+.DS_Store
+*.swo
+*.swn
+package-lock.json
```

Comment:
Please also include files and directories generated by "testem" such as but not limited to:
```
report.tap
coverage
.nyc_output
reports
instrumented
```

Reply:
there have been added.